src/framework/pages/admin/AbstractMainMenu.java

```diff
@@ -0,0 +1,93 @@
+package framework.pages.admin;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import framework.pages.admin.conferencerooms.ConferenceRoomPage;
+import framework.pages.admin.resources.ResourcesPage;
+import framework.selenium.SeleniumDriverManager;
+
+/**
+ * This class contains the main menu of Admin page
+ * @author Ruben Blanco
+ *
+ */
+public abstract class AbstractMainMenu {
+	protected WebDriver driver;	
+	protected WebDriverWait wait;
+	
+	@FindBy(linkText = "Room Manager") 
+	WebElement homeLink;
+
+	@FindBy(linkText = "Email Servers") 
+	WebElement emailServersLink;
+	
+	@FindBy(linkText = "Impersonation") 
+	WebElement impersonationLink;
+	
+	@FindBy(linkText = "Conference Rooms") 
+	WebElement conferenceRoomsLink;
+	
+	@FindBy(linkText = "Resources") 
+	WebElement resourcesLink;
+	
+	@FindBy(linkText = "Issues") 
+	WebElement issuesLink;
+	
+	@FindBy(linkText = "Locations") 
+	WebElement locationsLink;
+	
+	@FindBy(linkText = "Tablets") 
+	WebElement tabletsLink;
+	
+	/**
+	 * COnstructor
+	 */
+	public AbstractMainMenu() {
+		driver = SeleniumDriverManager.getManager().getDriver();
+		wait = SeleniumDriverManager.getManager().getWait();
+		PageFactory.initElements(driver, this);
+	}
+	
+	public HomeAdminPage clickEmailServersLink() {
+		emailServersLink.click();
+		return new HomeAdminPage();
+	}
+	
+	/**
+	 * This method only clicked this link 
```

Comment:
I suggest this remove this comment because you are not adding comments to the other methods